app/assets/stylesheets/base/_normalize.scss

```diff
@@ -0,0 +1,26 @@
+.some {
+  .bad {
+    .nesting {
+      .thats {
+        .sure {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5